ACE/rpmbuild/ace-tao.spec

```diff
@@ -1,7 +1,6 @@
 # Set the version number here.
 %define ACEVER  6.3.3
 %define TAOVER  2.3.3
-%define CIAOVER 1.3.3
 
```

Comment:
What about the bin/make_release.py script, that updates also this file and updates CIAOVER, any need for a change there?